packages/api-clients/open-api/inAppNotificationApi.yml

```
@@ -0,0 +1,329 @@
+openapi: 3.0.3
+info:
+  title: In App Notification Process Manager
+  description: This service is the in app notification manager
+  version: "{{version}}"
+  contact:
+    name: API Support
+    url: http://www.example.com/support
+    email: support@example.com
+  termsOfService: "http://swagger.io/terms/"
+  x-api-id: an x-api-id
+  x-summary: an x-summary
+servers:
+  - url: "/"
+    description: This service is the in app notification manager
+security:
+  - bearerAuth: []
+tags:
+  - name: notification
+    description: Implements in app notification manager
+    externalDocs:
+      description: Find out more
+      url: http://swagger.io
+  - name: health
+    description: Verify service status
+    externalDocs:
+      description: Find out more
+      url: http://swagger.io
+paths:
+  /notifications:
+    get:
+      tags:
+        - notification
+      summary: Get all notifications
+      description: Returns a list of all notifications for the requester
+      operationId: getNotifications
+      parameters:
+        - $ref: "#/components/parameters/CorrelationIdHeader"
+        - in: query
+          name: q
+          description: Query to filter notifications
+          schema:
+            type: string
+        - in: query
+          name: offset
+          required: true
+          schema:
+            type: integer
+            format: int32
+            minimum: 0
+        - in: query
+          name: limit
+          required: true
+          schema:
+            type: integer
+            format: int32
+            minimum: 1
+            maximum: 50
+      responses:
+        "200":
+          description: A list of notifications
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/Notifications"
+        "401":
+          description: Unauthorized
+          content:
+            application/problem+json:
+              schema:
+                $ref: "#/components/schemas/Problem"
+        "403":
+          description: Forbidden
+          content:
+            application/problem+json:
+              schema:
+                $ref: "#/components/schemas/Problem"
+  /notifications/bulk/markAsRead:
+    post:
+      security:
+        - Bearer: []
```

In other API specs, we have this; double-check it's present in all routes - or just remove it if not needed

```suggestion
      security:
        - bearerAuth: []
```